handsontable/src/plugins/formulas/utils.js

```diff
@@ -17,3 +21,74 @@ export function isEscapedFormulaExpression(expression) {
 export function unescapeFormulaExpression(expression) {
   return isEscapedFormulaExpression(expression) ? expression.substr(1) : expression;
 }
+
+/**
+ * Checks whether string looks like formula or not. Corresponds to {@link https://hyperformula.handsontable.com/api/globals.html#isformula|HyperFormula's implementation}.
+ *
+ * @param {string} value Checked value.
+ * @returns {boolean}
+ */
+export function isFormula(value) {
+  return typeof value === 'string' && value.startsWith('=');
+}
+
+/**
+ * Checks if provided value is a date according to cell meta.
+ *
+ * @param {*} value Checked value.
+ * @param {string} cellType Type of a cell.
+ * @returns {boolean}
+ */
+export function isDate(value, cellType) {
+  return typeof value === 'string' && cellType === 'date';
+}
+
+/**
+ * Checks if provided date is a valid date according to cell date format.
+ *
+ * @param {*} date Checked date.
+ * @param {object} dateFormat Handled format for a date.
+ * @returns {boolean}
+ */
+export function isDateValid(date, dateFormat) {
+  return moment(date, dateFormat, true).isValid();
+}
+
+/**
+ * Returns date formatted in HF's default format.
+ *
+ * @param {string} date Date formatted according to Handsontable cell date format.
+ * @param {string} dateFormat The format used for the date passed.
+ * @returns {string}
+ */
+export function getDateInHfFormat(date, dateFormat) {
+  return moment(date, dateFormat, true).format(DEFAULT_DATE_FORMAT_HYPERFORMULA);
+}
+
+/**
+ * Returns date formatted in HF's default format.
+ *
+ * @param {string} date Date formatted according to Handsontable cell date format.
+ * @param {string} dateFormat The format used for the date passed.
+ * @returns {string}
+ */
+export function getDateInHotFormat(date, dateFormat) {
```

Comment:
Can I ask to cover the `getDateInHotFormat ` util function with tests?

Reply:
Added init tests within: https://github.com/handsontable/handsontable/pull/10317/commits/f8de204b58316cd6649a69e8ea44f6685c3d8aed.